mobile/src/components/courses/components/course-card/components/vendor-label/vendor-label.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { FC } from 'react';
+
+import { View } from '~/components/common/common';
+import { vendorsKeyToImage } from '~/components/courses/components/course-card/common/maps/maps';
+
+import { styles } from './style';
+
+type Props = {
+  vendorKey: string;
+};
+
+const VendorLabel: FC<Props> = ({ vendorKey }) => {
+  const Image = vendorsKeyToImage[vendorKey.toLowerCase()];
```

Comment:
Isn't it already in lower case?

Reply:
Yes, already in lower. But I think, what if in the future key comes with the first capital letter) Useless think?

Reply:
it's not possible, all keys will be in snake case